util/http/http.go

```diff
@@ -18,8 +18,8 @@ import (
 const maxCookieLength = 4093
 
 // max number of chunks a cookie can be broken into. To be compatible with
-// widest range of browsers, we shouldn't create more than 30 cookies per domain
-var maxCookieNumber = env.ParseNumFromEnv(common.EnvMaxCookieNumber, 10, 0, 30)
+// widest range of browsers, you shouldn't create more than 30 cookies per domain
+var maxCookieNumber = env.ParseNumFromEnv(common.EnvMaxCookieNumber, 20, 0, math.MaxInt64)
```

Comment:
@zvlb Thanks for the PR!! Increasing the minimum count to 20 makes sense.

For the maximum count, I would continue to go with 30 to support all the browsers.

Reply:
The main thing I need from this pull request is to change the upper limit. 
To resolve this problem:
_err=the authentication token is 135509 characters long and requires 34 cookies but the max number of cookies is 30. Contact your Argo CD administrator to increase the max number of cookies_

Why should these constraints be hard-coded? Why can't the user setting up ArgoCD define this limit themselves?

Also, among all popular browsers, Opera has the minimum value for the number of cookies - it's not 30, it's 60.